.github/workflows/e2e.yml

```diff
@@ -7,6 +7,11 @@ env:
 on:
   pull_request:
     branches: [ "main" ]
+    paths-ignore:
+      - "**.md"
+      - "LICENSE"
+      - "docs/**"
+      - ".github/**"
```

Comment:
Why `".github/**"`? It will also ignore on PR with changes on github action pipelines.

Reply:
Yes, I'm not super sure about this one. I was thinking, that maybe it's not necessary to run `pull-nats-module-build` if only files in `.github` are changed. If the prow job doesn't run, then the github action also shouldn't run. 

You'd only have to uncomment this on your PR if you want to test changes to the action.

Reply:
lets remove it by default, if someone changes in `.github` then they would be bound to make everything green.